src/main/java/net/glowstone/GlowServer.java

```diff
@@ -1701,7 +1705,12 @@ public OfflinePlayer getOfflinePlayer(String name) {
     @Override
     public OfflinePlayer getOfflinePlayer(UUID uuid) {
         try {
-            return getOfflinePlayerAsync(uuid).get(5, TimeUnit.SECONDS); // todo: should we timeout? If yes, how long?
+            // probably blocking, timeout depending on config setting
+            if (config.getInt(Key.PROFILE_LOOKUP_TIMEOUT) <= 0) {
```

Comment:
Add a method in GlowServer to get this value (e.g. `getProfileLookupTimeout()`), with documentation specifying it is in seconds.